src/registry/children/index.stories.tsx

```diff
@@ -0,0 +1,507 @@
+import type {Meta, StoryObj} from '@storybook/react-vite';
+import {expect, userEvent, within} from 'storybook/test';
+
+import ValueDisplay from '@/registry/children/ValueDisplay';
+import type {ExtendedChildDetails} from '@/registry/children/types';
+import {withFormik} from '@/sb-decorators';
+
+import {FormioChildrenField} from './index';
+
+export default {
+  title: 'Component registry / special / children / presentation',
+  component: FormioChildrenField,
+  decorators: [withFormik],
+  args: {
+    componentDefinition: {
+      id: 'children',
+      type: 'children',
+      key: 'children',
+      label: 'Children',
+      enableSelection: false,
+    },
+  },
+} satisfies Meta<typeof FormioChildrenField>;
+
+type Story = StoryObj<typeof FormioChildrenField>;
+
+export const MinimalConfiguration: Story = {
+  parameters: {
+    formik: {
+      initialValues: {
+        children: [
+          {
+            bsn: '111222333',
+            firstNames: 'John Doe',
+            dateOfBirth: '2000-1-1',
+          },
+        ],
+      },
+    },
+  },
+};
+
+export const WithTooltipAndDescription: Story = {
+  args: {
+    componentDefinition: {
+      id: 'children',
+      type: 'children',
+      key: 'children',
+      label: 'Children',
+      enableSelection: false,
+      tooltip: 'Surprise!',
+      description: 'Description as help text',
+    },
+  },
+  parameters: {
+    formik: {
+      initialValues: {
+        children: [
+          {
+            bsn: '111222333',
+            firstNames: 'John Doe',
+            dateOfBirth: '2000-1-1',
+          },
+        ],
+      },
+    },
+  },
+};
+
+export const WithSelectionEnabled: Story = {
+  args: {
+    componentDefinition: {
+      id: 'children',
+      type: 'children',
+      key: 'children',
+      label: 'Children',
+      enableSelection: true,
+    },
+  },
+  parameters: {
+    formik: {
+      initialValues: {
+        children: [
+          {
+            bsn: '123456789',
+            firstNames: 'John',
+            dateOfBirth: '2000-1-1',
+          },
+          {
+            bsn: '074303909',
+            firstNames: 'Jane',
+            dateOfBirth: '1997-12-12',
+            selected: false,
+          },
+          {
+            bsn: '111222333',
+            firstNames: 'Jimmy',
+            dateOfBirth: '1995-4-3',
+            __addedManually: true,
+            __id: 'e47a2ed6-1f28-46c0-b6b9-bdb6d6d1dffd',
+            selected: true,
+          },
+        ],
+      },
+    },
+  },
+  play: async ({canvasElement, step}) => {
+    const canvas = within(canvasElement);
+
+    await step('Validate initial state', () => {
+      const checkboxes = canvas.getAllByRole('checkbox');
+      expect(checkboxes).toHaveLength(3);
+
+      // Only the selected child should be checked
+      expect(checkboxes[0]).not.toBeChecked();
+      expect(checkboxes[1]).not.toBeChecked();
+      expect(checkboxes[2]).toBeChecked();
```

Comment:
can we give the checkboxes a proper accessible label (use the child first name?) and use those in the queries here?